lib/jazzy/sourcekitten.rb

```diff
@@ -320,14 +320,18 @@ def self.deduplicate_declarations(declarations)
     def self.deduplication_key(decl)
       if decl.type.swift_extensible? || decl.type.swift_extension?
         [decl.usr, decl.name]
+      elsif decl.type.objc_class? || decl.type.objc_category?
+        name, = decl.objc_category_name || decl.name
+        [name, 'objc_class_and_categories']
```

Comment:
I like this solution — maybe a little bit of a hack, but much tighter than manually resolving the relationships!

I'd make the second array elem a symbol.